pkg/io/video/convert.go

```diff
@@ -87,6 +94,15 @@ func ToI420(r Reader) Reader {
 	})
 }
 
+func makeCopy(img image.YCbCr, cb, cr []uint8) image.YCbCr {
```

Comment:
I don't use a pointer here to use a stack allocated copy of the YCbCr structure and related slices. This should avoid adding garbage collection pressure to the cost of the copy.